lapma/db.php

```diff
@@ -3,7 +3,7 @@
 $server = "localhost";
 $user = "root";
 $password = "";
-$database = "lapma";
+$database = "yeolomco_lapma";
```

Comment:
this is local, did you do this for the Script?

Reply:
yes I changed the db name

Reply:
noted